projectile.el

```diff
@@ -1779,6 +1780,11 @@ This is a subset of `grep-read-files', where either a matching entry from
                (and ext (concat "*." ext)))))
       (or default-alias default-extension))))
 
+(defun projectile-grep-ignored-patterns ()
```

Comment:
I found `projectile-patterns-to-ignore` which returns list of `.projectile` entries without leading slash. But it doesn't used for grep ignoring.  This is feature or bug?

May be better name would be `projectile-grep-ignored` or `projectile-grep-find-ignored` (as  analog of `projectile-project-ignored` for grep/ag).